test/common/index.js

```diff
@@ -801,7 +801,14 @@ function hijackStdWritable(name, listener) {
 
   stream.writeTimes = 0;
   stream.write = function(data, callback) {
-    listener(data);
+    try {
+      listener(data);
+    } catch(e) {
+      process.nextTick(function() {
```

Comment:
I know you like `function`s but this could be a single line.
IMHO the indentation is weird